docs/source/articles/fulfillment-skus.html.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Fulfillment SKUs
+created_at: 2019/10/02
+excerpt: Define how products in your catalog are fulfilled. Fulfillment SKUs allow the automation of digital and similar items.
+---
+
+# Fulfillment SKUS
+
+Workarea v3.5 adds the concept of Fulfillment SKUs -- a model that defines policies for delivering ordered products to customers. This provides a more robust mechanism for the fulfillment of digital items while providing the flexibility to offer customized delivery behaviors at the SKU level.
```

Comment:
the term you're defining is plural, but the definition is singular; should make those consistent